internal/oauth/oauth.go

```diff
@@ -223,67 +223,7 @@ func (s *OAuthService) StartOAuthServer() (*TokenResponse, error) {
 			return
 		}
 
-		w.Header().Set("Content-Type", "text/html")
-		fmt.Fprintf(w, `
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Authentication Complete</title>
-    <style>
-        body {
-            font-family: Arial, sans-serif;
-            display: flex;
-            justify-content: center;
-            align-items: center;
-            height: 100vh;
-            margin: 0;
-            background-color: #f5f5f5;
-        }
-        .container {
-            text-align: center;
-            background-color: white;
-            padding: 40px;
-            border-radius: 8px;
-            box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
-        }
-        .check-circle {
-            width: 80px;
-            height: 80px;
-            background-color: #4CAF50;
-            border-radius: 50%%;
-            display: flex;
-            justify-content: center;
-            align-items: center;
-            margin: 0 auto 20px;
-        }
-        .check-mark {
-            color: white;
-            font-size: 40px;
-        }
-        h1 {
-            color: #333;
-            margin-bottom: 10px;
-        }
-        p {
-            color: #666;
-            font-size: 16px;
-        }
-    </style>
-</head>
-<body>
-    <div class="container">
-        <div class="check-circle">
-            <span class="check-mark">&#10004;</span>
-        </div>
-        <h1>Authentication complete</h1>
-        <p>You can close the tab and go back to the Hyphen CLI.</p>
-    </div>
-</body>
-</html>
-`)
-
+		http.Redirect(w, r, "https://hyphen.ai/cli?authenticate=true", http.StatusTemporaryRedirect)
```

Comment:
Thanks I copy pasta from the story card , will fix it 🙌🏽

Reply:
<img width="1350" alt="image" src="https://github.com/user-attachments/assets/c3334812-b8b9-4863-be45-e686777f5469">
for some reason any query param give me this

Reply:
@kylekampy

Reply:
The dev environment shows us what is coming for prod. Dev is looking at `authenticated` so that's what we should use. The actual returned UI between dev and prod is also different. Sounds like we'll deploy Monday next which will include this.

My assumption is that prod currently just has this placeholder page for the CLI. Probably worth double-checking with Nathan, given the discrepancy in the card between the title and body of it.